index.js

```diff
@@ -1 +1,44 @@
-document.write('<script src="js/index.js"></script>');
+// Get the elements for the pop-up menu and the union-bar
+const popUpSection = document.querySelector('.pop-up-section');
+const unionBar = document.querySelector('.union-bar');
+const crossBtn = document.querySelector('.cross-btn');
+const popUpItems = document.querySelectorAll('.pop-menu-item');
+
+// Function to toggle the visibility of the pop-up menu
+function togglePopUp() {
+  popUpSection.classList.toggle('show');
+}
+
+// Add a click event listener to the union-bar
+unionBar.addEventListener('click', togglePopUp);
+
+// Function to hide the pop-up menu when a menu item is clicked
+function hidePopUp() {
+  popUpSection.classList.remove('show');
+}
+
+// Add a click event listener to each pop-up menu item
+popUpItems.forEach((item) => {
+  item.addEventListener('click', hidePopUp);
+});
```

Comment:
- [ ] I think it would be great if you fix the problem when the user press clicks (or taps) any of the mobile menu options, the mobile menu disappears. since when I press on the option it shows the correct part of the page but it does not disappear so kindly work on this part and try to fix it ⭐💯✨💪🏻 kindly check the video I included above. ⤴